### ISSUE TITLE
In v3.0 format the metadata is not shown properly

### ISSUE THREAD

**Issue by rubinatorz** [mitre-attack/attack-navigator#188]
Hi @isaisabel 

I'm working on our DeTT&CT project to add support for the new ATT&CK sub-techniques. Within DeTT&CT we are using the metadata property of the techniques object heavily. I was testing the metadata support in the new beta Navigator at:
https://mitre-attack.github.io/attack-navigator/beta/enterprise/

And I noticed that the metadata is not shown properly:

<img width="337" alt="Screenshot 2020-06-29 at 16 44 25" src="https://user-images.githubusercontent.com/11735227/86020128-cc29be00-ba27-11ea-8f1e-bb5d076dd64b.png">

The JSON of the layer template looks like this:
```
{
	"name": "layer1",
	"version": "3.0",
	"domain": "mitre-enterprise",
	"description": "",
	"filters": {
		"stages": [
			"act"
		],
		"platforms": [
			"Windows",
			"Linux",
			"macOS"
		]
	},
	"sorting": 0,
	"layout": {
		"layout": "side",
		"showID": false,
		"showName": true
	},
	"hideDisabled": false,
	"techniques": [
		{
			"techniqueID": "T1212",
			"tactic": "credential-access",
			"color": "#74c476",
			"comment": "",
			"enabled": true,
			"metadata": [],
			"showSubtechniques": false,
			"metadata": [{"name": "Rating", "value": "This is real cool stuff!"}]
		}
	],
	"gradient": {
		"colors": [
			"#ff6666",
			"#ffe766",
			"#8ec843"
		],
		"minValue": 0,
		"maxValue": 100
	},
	"legendItems": [],
	"metadata": [],
	"showTacticRowBackground": false,
	"tacticRowBackground": "#dddddd",
	"selectTechniquesAcrossTactics": true,
	"selectSubtechniquesWithParent": true
}
```

Am I doing something wrong or is this still under construction?

Thanks!

Regards,
Ruben Bouman

**Comment by isaisabel** [comment]
Hi @rubinatorz,

This is a known bug first reported in #153 and fixed in PR #159. This bugfix will be released alongside version 3.1 of the Navigator early next month, but you're welcome to check it out before the release if you'd like (the branch is [here](https://github.com/mitre-attack/attack-navigator/tree/feature/subtechniques-3.1), and the changelog [here](https://github.com/mitre-attack/attack-navigator/blob/feature/subtechniques-3.1/CHANGELOG.md)).

**Comment by rubinatorz** [comment]
Hi @isaisabel,

Thanks for getting back on this. I checked the open issues, but I didn't check the closed issues. So sorry for this duplicate issue. Good to hear that this is fixed! I just tried the feature/subtechniques-3.1 branch and indeed the issue is fixed.

I see that the metadata values are now aligned with each other and the line height of each line is increased:
<img width="366" alt="Screenshot 2020-06-29 at 17 13 00" src="https://user-images.githubusercontent.com/11735227/86023478-039a6980-ba2c-11ea-940a-33134afd8aaa.png">

In the previous version this was:
<img width="286" alt="Screenshot 2020-06-29 at 17 12 51" src="https://user-images.githubusercontent.com/11735227/86023522-1319b280-ba2c-11ea-8854-15598e8d255e.png">

I can imagine the choice for this, but in our case it just looks a little less nice in the new version of the metadata. Is there a change of having an option to influence this behaviour?

Thanks!
